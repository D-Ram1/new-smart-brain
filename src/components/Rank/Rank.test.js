import React from 'react';
import { shallow } from 'enzyme';
import Rank from './Rank';

it('renders without crashing', () => {
    expect(shallow(<Rank />)).toMatchSnapshot();
})