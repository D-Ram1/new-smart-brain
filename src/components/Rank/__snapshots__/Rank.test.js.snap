// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <div
    className="white f3"
  >
    Hi undefined! Your current entry count is...
  </div>
  <div
    className="white f1"
  />
</div>
`;
